### PR TITLE
HOMS-155 Add profiles for order list display settings

### PR DESCRIPTION
**Pull request by TimAle**

### CODE REVIEW

**Comment by sugarfree1** [comment]
Process "active" css-class for select options.
Fix indent for select.
Fix underline for select.

**Comment by TimAle** [comment]
UPD

File: app/controllers/api/v1/profiles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      end
+
+      def replace_user_email!(params)
+        user_id = User.id_from_email params[:user_email]
+        replace!(params, :user_email, :user_id, user_id)
+      end
+
+      def replace_order_type_code!(params)
+        order_type_id = OrderType.id_from_code params[:order_type_code]
+        fail ActiveRecord::RecordNotFound unless order_type_id
```
Comment (sugarfree1) [inline app/controllers/api/v1/profiles_controller.rb]: You use ```record_not_found``` in ```replace!```. I think this is the same. Choose an option.

File: app/views/orders/index.html.haml [hunk 2]
```diff
@@ -1,24 +1,16 @@
 - content_for(:title, t('.title'))
 %h2= t('.title')
 = render partial: 'search_form'
+
+- with_custom_profile = list_filter.filter_values[:order_type_id].present?
```
Comment (sugarfree1) [inline app/views/orders/index.html.haml]: You use this var only when ```@orders.present?```. Evaluate it under this condition.

**Comment by sugarfree1** [comment]
There is still a chance to hide order code.

**Comment by TimAle** [comment]
UPD

**Comment by TimAle** [comment]
UPD

File: app/assets/javascripts/components/order_list.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    })
+  },
+
+  updateProfile() {
+    $.ajax('/profiles/' + this.state.profile.id, {
+      method: 'PUT',
+      dataType: 'json',
+      contentType: 'application/json',
+      data: JSON.stringify(this.state.profile)
+    })
```
Comment (sugarfree1) [inline app/assets/javascripts/components/order_list.jsx]: Don’t you need `success` callback here?

File: config/locales/en.yml [hunk 4]
```diff
@@ -251,6 +251,7 @@ en:
     select2:
       not_set: Empty
     user_not_found: Current user not found in Activiti
+    columns_settings: Column settings
```
Comment (sugarfree1) [inline config/locales/en.yml]: Key doesn’t match value.

File: config/locales/en.yml [hunk 5]
```diff
@@ -296,3 +297,10 @@ en:
         company: Company name
         department: Department name
         password: Password
+    profiles:
+      resource: Profiles
```
Comment (sugarfree1) [inline config/locales/en.yml]: Add translations for :ru.

Comment (TimAle) [reply config/locales/en.yml]: Are you sure? I thought we have no russian docs for API.

**Comment by TimAle** [comment]
UPD

**Pull request merged**